pkg/apis/eventing/v1alpha1/condition_status_type.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ConditionType string
+
+const (
+	// ConditionReady specifies that the resource is ready.
+	ConditionReady ConditionType = "Ready"
+)
+
+// ConditionStatus describes the state of a resource at a point in time.
+type ConditionStatus struct {
+	// Type of condition.
+	// +required
+	Type ConditionType `json:"type"`
+
+	// Status of the condition, one of True, False, Unknown.
+	// +required
+	Status corev1.ConditionStatus `json:"status"`
+
+	// The last time this condition was updated.
+	// +optional
+	LastUpdateTime metav1.Time `json:"lastUpdateTime,omitempty"`
+
+	// Last time the condition transitioned from one status to another.
+	// +optional
+	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty"`
```

Comment:
Why do we have this? Presumably, `LastUpdateTime` is used to signal liveness of the controller.

Reply:
I took the version from Bus, Bus had this. I removed it in the common pkg version I am pushing https://github.com/knative/pkg/pull/66